Arriba/Arriba.Test/Model/ColumnFactoryTests.cs

```diff
@@ -16,6 +15,12 @@ public class ColumnFactoryTests
     {
         private string _defaultWrappingFormatString = "UntypedColumn<{0}>;FastAddSortedColumn<{0}>;ValueTypeColumn<{0}>";
 
+        [TestCleanup]
+        public void TestCleanup()
+        {
+            ColumnFactory.ResetColumnCreators();
```

Comment:
This method was only created for test contexts.  The idea that column creation is transient is unpleasant.  For now I'd like to avoid that.